<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SdNdeHalcWidget</class>
 <widget class="QWidget" name="C_SdNdeHalcWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C_SdNdeHalcWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>25</number>
   </property>
   <property name="topMargin">
    <number>25</number>
   </property>
   <property name="rightMargin">
    <number>25</number>
   </property>
   <property name="bottomMargin">
    <number>25</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="pc_LayoutImportExport">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pc_PubImportConfig">
       <property name="text">
        <string>-Import Config</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pc_PubExportConfig">
       <property name="text">
        <string>-Export Config</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pc_PubSettings">
       <property name="text">
        <string>-Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="pc_LayoutActions">
     <item>
      <widget class="stw_opensyde_gui_elements::C_OgeLabGroupItem" name="pc_LabFileTitle">
       <property name="text">
        <string>-Hardware Description File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="stw_opensyde_gui_elements::C_OgeLabGroupItemValue" name="pc_LabFileCurrent">
       <property name="text">
        <string>-my_description.xyz</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pc_PubMagic">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 255);</string>
       </property>
       <property name="text">
        <string>Do the MAGIC!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pc_PubCleanUp">
       <property name="text">
        <string>-Clean Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pc_PubLoad">
       <property name="text">
        <string>-Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pc_PubViewDetails">
       <property name="text">
        <string>-View Details</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="pc_LayoutContent" stretch="0,1,1">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="pc_LayoutOverview">
       <item>
        <widget class="QPushButton" name="pc_PubOverview">
         <property name="text">
          <string>-Overview</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="stw_opensyde_gui::C_SdNdeHalcChannelTreeView" name="pc_TreeChannels"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="pc_WiOverviewTable" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>work in progress</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="stw_opensyde_gui::C_SdNdeHalcChannelWidget" name="pc_WiChannelEdit" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabGroupItem</class>
   <extends>QLabel</extends>
   <header>C_OgeLabGroupItem.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabGroupItemValue</class>
   <extends>QLabel</extends>
   <header>C_OgeLabGroupItemValue.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui::C_SdNdeHalcChannelTreeView</class>
   <extends>QTreeView</extends>
   <header>C_SdNdeHalcChannelTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui::C_SdNdeHalcChannelWidget</class>
   <extends>QWidget</extends>
   <header>C_SdNdeHalcChannelWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
